test/halmos/HalmosTest.sol

```diff
@@ -74,17 +95,9 @@ contract HalmosTest is SymTest, Test {
             address borrower = svm.createAddress("borrower");
             args = abi.encode(marketParams, borrower, assets, shares, emptyData);
         } else if (selector == morpho.flashLoan.selector) {
-            uint256 rand = svm.createUint256("rand");
-            address token;
-            if (rand == 0) {
-                token = address(loanToken);
-            } else if (rand == 1) {
-                token = address(collateralToken);
-            } else {
-                ERC20Mock otherToken = new ERC20Mock();
-                token = address(otherToken);
-            }
-            args = abi.encode(marketParams, token, assets, emptyData);
+            address token = svm.createAddress("token");
+            bytes memory _data = svm.createBytes(1024, "_data");
+            args = abi.encode(token, assets, _data);
```

Comment:
non-empty `_data` is needed for FlashBorrowerMock.onMorphoFlashLoan().

Reply:
I see that it expects `(address assets, bytes data)` as argument. So what happens if it is empty ? Does it mean that the call revert, and so Halmos considers it as a revert path in the verification ?

Reply:
yes. if data is empty, then onMorphoFlashLoan will revert due to `abi.decode(data, (address))`. as a result, flashLoan() will always revert. since we have `vm.assume(success);`, all flashLoan() paths will be silently ignored. essentially, this causes flashLoan() to be omitted from the verification.